### PR TITLE
Groceries/fixes

### PR DESCRIPTION
**Pull request by Geno22**
Done here :

- Fixed grocery item cross background
- Checked that everything is deleted when removing a house
- Changed the shoplist Firestore creation to happen when the house is created

### CODE REVIEW

File: app/src/main/java/com/github/houseorganizer/houseorganizer/panels/household/EditHouseholdActivity.java [hunk 1]
```diff
@@ -354,8 +362,8 @@ public void deleteTaskList(View view) {
                 .addOnFailureListener(tlDeletionFailed);
     }
 
-    public void deleteCalendar(View view) {
-        firestore.collection("events")
+    public Task<QuerySnapshot> deleteCalendar(View view) {
```
Comment (Oxydeas) [inline app/src/main/java/com/github/houseorganizer/houseorganizer/panels/household/EditHouseholdActivity.java:365]: That can be useful to wait in the tests

**Review by Oxydeas — APPROVED** [review]
Thanks for the fix!

File: app/src/main/res/layout/shoplist_row.xml [hunk 2]
```diff
@@ -9,17 +9,17 @@
...
     <ImageButton
         android:id="@+id/delete_item_button"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_weight="0.1"
+        android:layout_weight="8"
         android:adjustViewBounds="true"
-        android:background="@color/common_google_signin_btn_text_dark"
+        android:background="#00FFFFFF"
```
Comment (Arkadyus) [inline app/src/main/res/layout/shoplist_row.xml:22]: If i recall correctly you can use "@android:color/transparent" to have a transparent background instead

**Review by Arkadyus — CHANGES_REQUESTED** [review]
Nice fixes, thanks for the PR ! Just a tiny change because im picky

File: app/src/main/res/layout/shoplist_row.xml (hunk 2 above)
Comment (Geno22) [reply app/src/main/res/layout/shoplist_row.xml]: Done

**Review by Arkadyus — APPROVED** [review]
Perfect, thanks for the changes !

**Pull request merged**